zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/ErrorMessages.java

```diff
@@ -23,4 +23,6 @@ public final class ErrorMessages {
   public static final String ERROR_UNKNOWN_SORT_ORDER = "Unknown sortOrder: %s";
   public static final String ERROR_SEARCH_BEFORE_AND_AFTER =
       "Both searchAfter and searchBefore cannot be set at the same time";
+  public static final String ERROR_MESSAGE_AT_LEAST_ONE_FIELD =
+      "At least one field between %s is required";
```

Comment:
🔧 Just a gut feeling, but "At least one of %s is required" sounds a bit cleaner.